examples/defender-test-project/serverless.yml

```diff
@@ -112,6 +112,24 @@ resources:
           url: ${self:custom.config.notifications.slack}
         paused: false
 
+    categories:
+      high-severity:
```

Comment:
if user has SSOT would it remove the high-severity category? Curious on whether or not we are mandating that these categories exist or if they are optional

Reply:
Good question! With SSOT enabled, removing these default categories from the YAML, and deploying will also remove it from Defender.

That being said, I don't think we _should_ require these categories to exists (i.e. they can't delete them). The objective should be to have default categories so it's 1 less click, and it incentivises to configure them. What do you think?

Reply:
my personal preference would be to allow them to be deleted, but may be good to check with @pasevin that this aligns with the product intent or if these three categories should always exist by the "guide the user into good practices" logic

Reply:
Removed the create and delete capabilities for notification categories.